src/content/docs/alerts-applied-intelligence/transition-guide.mdx

```diff
@@ -136,7 +136,7 @@ Here is a summary of what we plan to EOL in 2022 and 2023, why this is happening
     id="incident-response"
     title="September 30, 2022: Native experience in New Relic to replace classic incident response workflow."
   >
-    **Reason for EOL:** The classic [incident response workflow](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents/) involves the **Incident** and **Violation details** pages in the UI. It also uses APIs for incidents and violations.
+    **Reason for EOL:** The classic [incident response workflow](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents/) involves the **Incident** and **Violation details** pages in the UI. It also uses APIs for incidents and issues.
```

Comment:
Refers to the classic incident response workflow, which works with incidents and violations -- roll back this change.